### PR TITLE
Displaying the UI for the upload feature

### PR DESCRIPTION
**Pull request by ArinolaMordi**
Description
On this branch I worked on displaying the UI features for where the user would be inputting in all their information for it to be collected and sent to the database. 
Milestones
This branch worked towards Technical Challenge 2 where I would be displaying a map view of the projects submitted by each Tiny Thinker. On the map, there would be different markers that show the project name, description and image of project submitted when clicked ! This is where the project name, image, description and location would be inputted 

Test Plan
Attached below is a picture of  the UI 
Screenshot 2023-07-31 at 4 09 51 PM

### CODE REVIEW

File: my-app/src/App.js [hunk 1]
```diff
@@ -24,33 +21,14 @@ function App() {
...
 
   useEffect(() => {
-    // Save the user data to storage whenever the user state changes
     localStorage.setItem("user", JSON.stringify(user));
   }, [user]);
 
   const [watchedVideosCount, setWatchedVideosCount] = useState(0);
-  const [ttvideos, setTTVideos] = useState([]);
-
-  useEffect(() => {
```
Comment (nokolo99) [inline my-app/src/App.js:34]: What is this code you took away? I can't tell what it was doing before

**Comment by nokolo99** [comment]
Overall looks good, you just took away some code that I was not sure about. Could you explain why you removed those lines of code?

Also there seem to be merge conflicts so you will have to solve those manually before moving forward!

File: my-app/src/App.js (hunk 1 above)
Comment (ArinolaMordi) [reply my-app/src/App.js:34]: Yeah it wasn't doing anything there when I took a clear look at it . I think I was trying to implement a feature and forgot to take out that code.

Comment (nokolo99) [reply my-app/src/App.js:34]: Sounds good!

**Pull request merged**